src/cases/components/restricted-case-access/restricted-case-access.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RestrictedCaseAccessComponent } from './restricted-case-access.component';
+
+describe('RestrictedCaseAccessComponent', () => {
+  let component: RestrictedCaseAccessComponent;
+  let fixture: ComponentFixture<RestrictedCaseAccessComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [],
+      declarations: [RestrictedCaseAccessComponent]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
```

Comment:
@johnbenjamin-hmcts - is there a reason why there are two `beforeEach()` blocks? Can they be combined?

Reply:
Thank you @Daniel-Lam, I have done the changes as you suggested.